WordPress/src/main/java/org/wordpress/android/ui/themes/ThemeBottomSheetFragment.kt

```diff
@@ -0,0 +1,78 @@
+package org.wordpress.android.ui.themes
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.View.INVISIBLE
+import android.view.View.VISIBLE
+import android.view.ViewGroup
+import androidx.annotation.NonNull
+import androidx.lifecycle.ViewModelProvider
+import com.google.android.material.bottomsheet.BottomSheetBehavior.STATE_EXPANDED
+import com.google.android.material.bottomsheet.BottomSheetBehavior.from
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import org.wordpress.android.R
+import org.wordpress.android.WordPress
+import org.wordpress.android.databinding.ThemeActivationBottomSheetBinding
+import org.wordpress.android.viewmodel.themes.ThemesViewModel
+import org.wordpress.android.viewmodel.themes.ThemesViewModel.BottomSheetUIState.Selection.KeepCurrentHomepage
+import org.wordpress.android.viewmodel.themes.ThemesViewModel.BottomSheetUIState.Selection.UseThemeHomepage
+import javax.inject.Inject
+
+class ThemeBottomSheetFragment : BottomSheetDialogFragment() {
+    @Inject lateinit var viewModelFactory: ViewModelProvider.Factory
+    private lateinit var viewModel: ThemesViewModel
+
+    companion object {
+        const val TAG = "ThemeActivationBottomSheetFragment"
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        return inflater.inflate(R.layout.theme_activation_bottom_sheet, container)
```

Comment:
It would be nice to align the layout filename with the [new Fragment name](https://github.com/wordpress-mobile/WordPress-Android/pull/15043/files#r675377720).

Reply:
Done